crates/ibc/src/core/ics04_channel/msgs/chan_open_try.rs

```diff
@@ -56,17 +56,28 @@ impl TryFrom<RawMsgChannelOpenTry> for MsgChannelOpenTry {
             .channel
             .ok_or(ChannelError::MissingChannel)?
             .try_into()?;
+
+        chan_end_on_b.verify_state_matches(&State::TryOpen)?;
+
+        #[allow(deprecated)]
+        if !raw_msg.previous_channel_id.is_empty() {
```

Comment:
Not that we have this check, we no longer need to store the `previous_channel_id` (since we ensure that it is `""`)

Reply:
Can you point me to the place this check is done?
Also, I was wondering; from a users' perspective, who just wanted to perform this convestion on a `RawMsgChannelOpenTry`, Shouldn't we keep this, not allowing them to end up with a worng previous_channel_id?

Reply:
I'm just saying that [this field](https://github.com/cosmos/ibc-rs/blob/1134f01f1ce926a22c8a899d60b70eab6b70966b/crates/ibc/src/core/ics04_channel/msgs/chan_open_try.rs#L35) can be removed, since we now ensure that it is always an empty string (so storing the empty string is redundant).

> Shouldn't we keep this, not allowing them to end up with a worng previous_channel_id?

Yes, we should keep the check

Reply:
ty for clarifying [7f65497](https://github.com/cosmos/ibc-rs/pull/650/commits/7f65497c1cb727c584092b439e46026981045468)